docs/from_bosh_to_kube.md

```diff
@@ -307,6 +307,8 @@ instance_groups:
           # ImagePullSecrets is an optional list of references to secrets to use for pulling any of the images.
           # This field in PodSpec can be automated by setting the imagePullSecrets in a serviceAccount.
           ImagePullSecrets: {}
+          # Tolerations and taints is a concept defined in kubernetes to repel pods from nodes. [4]
```

Comment:
`Tolerations and taints are a concept defined in kubernetes`

Reply:
Done.